### PR TITLE
tmux: mouse support

### PR DESCRIPTION
**Pull request by cor**
### Description

Allows you to enable mouse support in tmux

### Checklist

- [x] Change is backwards compatible.

- [x] Code formatted with `./format`.

- [x] Code tested through `nix-shell --pure tests -A run.all`.

- [x] Test cases updated/added. See [example](https://github.com/nix-community/home-manager/commit/f3fbb50b68df20da47f9b0def5607857fcc0d021#diff-b61a6d542f9036550ba9c401c80f00ef).

- [x] Commit messages are formatted like

    ```
    {component}: {description}

    {long description}
    ```

### CODE REVIEW

File: modules/programs/tmux.nix [hunk 1]
```diff
@@ -204,6 +205,14 @@ in {
         description = "VI or Emacs style shortcuts.";
       };
 
+      mouse = mkOption {
+        default = false;
+        example = true;
+        description = ''
+          Mouse support.
+        '';
+      };
```
Comment (berbiche) [inline modules/programs/tmux.nix:214]: This can be simplified:
```suggestion
      mouse = mkEnableOption "mouse support";
```

**Review by berbiche — CHANGES_REQUESTED** [review]
Thanks for the contribution!

File: modules/programs/tmux.nix (hunk 1 above)
Comment (cor) [reply modules/programs/tmux.nix:214]: Ah this is much nicer, thanks!

**Comment by berbiche** [comment]
Hi,
Can you update the tmux tests to reflect your change?
Thanks

**Pull request closed without merging**